charts/xrd-vrouter/values.yaml

```diff
@@ -239,12 +239,11 @@ mgmtInterfaces: []
 # More information about the CPU behavior of XRd vRouter can be found
 # here: @@@
```

Comment:
While we are at it I think let's get rid of this line which has been here since day 1 :)

Might be worth checking with Paul if there is any appropriate external documentation that we can link to here (I suspect not?)

Reply:
There is not documentation, so I've removed the whole comment.